update(perf): Update performance landing and widget descriptions (#4500)
getsentry/sentry-docs: src/docs/product/performance/widgets.mdx

## Most Improved

Shows the most improved transactions and visualizes the improvement for the selected transaction in the list. Only trends of a specific significance and throughput are shown. Learn more in [Trends](/product/performance/trends).

## Most Regressed

Shows the most regressed transactions and visualizes the regression for the selected transaction in the list. Only trends of a specific significance and throughput are shown. Learn more in [Trends](/product/performance/trends).

## Most Related Issues

Shows the highest count of any issue group related to a transaction and visualizes the issue count over time.

## Most Slow Frames

Shows transactions with the most slow frames for mobile projects. Learn more in [Mobile Vitals](/product/performance/mobile-vitals).

## Most Frozen Frames

Shows transactions with the most frozen frames for mobile projects. Learn more in [Mobile Vitals](/product/performance/mobile-vitals).

## Worst LCP Vitals

Shows transactions with the highest count of poor LCP (Largest Contentful Paint) and visualizes the proportions of user experience for this vital. Learn more in [Web Vitals](/product/performance/web-vitals).

## Worst FCP Vitals

> Shows transactions with the highest count of poor FCP (First Contentful Paint) and visualizes the split in vital performance to illustrate the proportions of user experience. Learn more in [Web Vitals](/product/performance/web-vitals).

Reviewer: ```suggestion
Shows transactions with the highest count of poor FCP (First Contentful Paint) and visualizes the proportions of user experience for this vital. Learn more in [Web Vitals](/product/performance/web-vitals).
```

## Worst FID Vitals

> Shows transactions with the highest count of poor FID (First Input Delay) and visualizes the split in vital performance to illustrate the proportions of user experience. Learn more in [Web Vitals](/product/performance/web-vitals).

Reviewer: ```suggestion
Shows transactions with the highest count of poor FID (First Input Delay) and visualizes the proportions of user experience for this vital. Learn more in [Web Vitals](/product/performance/web-vitals).
```

> ## Slow HTTP Ops
> 
> Shows transactions with slow cumulative HTTP spans. Slow HTTP Ops are note shown for all projects and transactions as the transactions must contain HTTP spans. Check out the SDK documentation for the project in question for more information.
> 
> ## Slow DB Ops
> 
> Shows transactions with slow cumulative DB spans. Slow DB Ops are not shown for all projects and transactions as the transactions must contain DB spans. Check out the SDK documentation for the project in question for more information.

Reviewer: Just changing the capitalization since we're talking about a general concept here and not a UI element.

```suggestion
## Slow HTTP Ops

Shows transactions with slow cumulative HTTP spans. Slow HTTP ops are note shown for all projects and transactions as the transactions must contain HTTP spans. Check out the SDK documentation for the project in question for more information.

## Slow DB Ops

Shows transactions with slow cumulative DB spans. Slow DB ops are not shown for all projects and transactions as the transactions must contain DB spans. Check out the SDK documentation for the project in question for more information.
```

## Most Slow Frames

Shows transactions with the most slow frames for mobile projects. See [Mobile Vitals](/product/performance/mobile-vitals) for more information.

## Most Frozen Frames

Shows transactions with the most frozen frames for mobile projects. See [Mobile Vitals](/product/performance/mobile-vitals) for more information.

## Worst LCP Vitals

Shows transactions with the highest count of poor LCP (Largest Contentful Paint), and visualizes the split in vital performance to illustrate the proportions of user experience. See [Web Vitals](/product/performance/web-vitals) for more information.

## Worst FCP Vitals

Shows transactions with the highest count of poor FCP (First Contentful Paint), and visualizes the split in vital performance to illustrate the proportions of user experience. See [Web Vitals](/product/performance/web-vitals) for more information.

## Worst FID Vitals

Shows transactions with the highest count of poor FID (First Input Delay), and visualizes the split in vital performance to illustrate the proportions of user experience. See [Web Vitals](/product/performance/web-vitals) for more information.

## Slow HTTP Ops

> Show transactions with slow cumulative HTTP spans. Slow HTTP Ops will not be shown for all projects and transactions as the transactions must contain HTTP spans. See the sdk documentation for the project in question for more information.

Author: It's a general term, slow X ops means it sorts by the total number of HTTP time within a transaction, hence where the cumulative comes from. Maybe `slow cumulative HTTP span duration`? I'm not sure how to clear this up, but basically it should show you transactions with (mostly) slow HTTP, if we're recording it for their sdk.

> Show transactions with slow cumulative HTTP spans. Slow HTTP Ops will not be shown for all projects and transactions as the transactions must contain HTTP spans. See the sdk documentation for the project in question for more information.

Reviewer: Okay, got it. I only wanted to clarify so I could figure out how to format this. Sorry I should have made that clear. The actual wording of the definition is fine.

## Slow DB Ops

> Show transactions with slow cumulative DB spans. Slow DB Ops will not be shown for all projects and transactions as the transactions must contain DB spans. See the sdk documentation for the project in question for more information.

Reviewer: ```suggestion
Shows transactions with slow cumulative DB spans. Slow DB Ops are not shown for all projects and transactions as the transactions must contain DB spans. Check out the SDK documentation for the project in question for more information.
```

## Slow Browser Ops

> Show transactions with slow cumulative Browser spans. These show frontend transactions with poor performing browser spans, such as DNS lookup, TLS/SSL, etc.

Reviewer: What is "these" at the beginning of the second sentence here? The widgets?

```suggestion
Shows transactions with slow cumulative browser spans. These show frontend transactions with poor performing browser spans, such as DNS lookup, TLS/SSL, and so on.
```

## Slow Resource Ops

> Show transactions with slow cumulative Resource spans. These show frontend transactions with poor performing resources spans, and show cumulative time spent loading scripts, CSS, etc.

Reviewer: ```suggestion
Shows transactions with slow cumulative resource spans. These show frontend transactions with poor performing resources spans, and show cumulative time spent loading scripts, CSS, and so on.
```
